public/jsx/views/card-listing.jsx

```diff
@@ -23,25 +23,16 @@ module.exports = React.createClass({
   },
 
   componentDidMount: function() {
-    $.ajax({
-      method: 'get',
-      url: '/api/braintree/mozilla/paymethod/',
-      context: this,
-    }).then(function(data) {
-      if (this.isMounted()) {
-        // Ignore react/no-did-mount-set-state eslint warning
-        if (data.length) {
-          console.log('Card data found, showing card form');
-          this.setState({cards: data}); // eslint-disable-line
-        } else {
-          console.log('No card data found, showing card form');
-          this.transitionTo('card-form');
-        }
-      }
-    }).fail(function() {
-      // TODO: some error state.
-      console.log('Card list retrieval failed');
-    });
+    var user = UserStore.getLoggedInUser();
```

Comment:
i'd recommend one of them, the actions and stores are extremely easy to test in isolation. and they are designed so that a flux instance is passed down from the top of the heirarchy all the way to the bottom. so when you're testing an individual component, you just have to make a fake flux instance. or when you have a component connect to a store, you just have to pass in primitive props.